### DISCUSSION TITLE
How to enable Ethernet if disable and no access to SSH

### DISCUSSION THREAD

**Discussion by lutfor-diu** [MichaIng/DietPi#6531] [Q&A]
Hi,
Recently i came across a wired situation. where i accidentally disable ethernet adapter from dietpi-config setting. and i lost SSH access due to Lan port disabled.  In this situation i was unable to re enable ethernet and gain Access to SSH. So i had no option but setup fresh dietpi. 

is there any option to re-enable ethernet in that situation? like editing MMC/SD card file. or config file in memory card?

**Comment by Joulinar** [comment]
You would need to have another Linux system to be able to access the ext4 partition. There you could have a look to `/etc/network/interfaces`.

**Reply by lutfor-diu** [reply]
what would be setting to enable ethernet? in `/etc/network/interfaces`

**Reply by Joulinar** [reply]
Following should be enabled

```
iface eth0 inet static
```

**Reply by MichaIng** [reply]
And above that one, take care to remove the leading hash `#` in front if this line:
```sh
allow-hotplug eth0
```

**Reply by lutfor-diu** [reply]
thank you so much
